src/plugins/workspace/public/plugin.ts

```diff
@@ -124,12 +145,8 @@ export class WorkspacePlugin implements Plugin<{}, {}> {
           });
         })();
 
-        // If workspace is enabled and user has entered workspace, hide advance settings and dataSource menu and disable
-        const managementSectionApps = management.sections.section.opensearchDashboards.getAppsEnabled();
-        const disabledApps = managementSectionApps.filter(
-          (app) => app.id === 'settings' || app.id === 'dataSources'
-        );
-        disabledApps?.forEach((app) => app.disable());
+        //  Hide advance settings and dataSource menus and disable in setup
+        this.disableManagementApps(core, management);
```

Comment:
Nit: would you mind moving this logic out of the `else` and put it into `setup`?

Reply:
Thanks for reminding, i forgot this.